### ISSUE TITLE
[bicep] CKV_AZURE_14 false positive

### ISSUE THREAD

**Issue by holly79** [bridgecrewio/checkov#2871]
**Describe the issue**
httpsOnly is set to true, but the ckeck failed

**Examples**
```bicep
resource Site 'Microsoft.Web/sites@2021-03-01' = {
  name: _siteName
  kind: kind
  location: location
  tags: _tags
  properties: {
    serverFarmId: ServerFarm.id
    httpsOnly: true
    clientAffinityEnabled: true 
    reserved: _isReserved
    siteConfig: {
      minTlsVersion: '1.2'
      scmMinTlsVersion: '1.2'
      http20Enabled: true
      ftpsState: 'FtpsOnly'
    }
  }
  identity: {
    type: 'SystemAssigned'
  }
}
```

**Version (please complete the following information):**
 - Checkov Version [2.0.1084]

**Additional context**

**Comment by gruebel** [comment]
hey @holly79 I run it locally and `CKV_AZURE_14` passes for me, but of course `CKV_AZURE_17` failed, which is correct. Do you maybe have some other resources inside the file, which could be of some interest? Maybe you accidentally added multiple `resource Site` inside the template file?

```shell
bicep scan results:

Passed checks: 4, Failed checks: 1, Skipped checks: 0

Check: CKV_AZURE_15: "Ensure web app is using the latest version of TLS encryption"
        PASSED for resource: Microsoft.Web/sites.Site
        File: example.bicep:1-21
        Guide: https://docs.bridgecrew.io/docs/bc_azr_networking_6
Check: CKV_AZURE_14: "Ensure web app redirects all HTTP traffic to HTTPS in Azure App Service"
        PASSED for resource: Microsoft.Web/sites.Site
        File: example.bicep:1-21
        Guide: https://docs.bridgecrew.io/docs/bc_azr_networking_5
Check: CKV_AZURE_16: "Ensure that Register with Azure Active Directory is enabled on App Service"
        PASSED for resource: Microsoft.Web/sites.Site
        File: example.bicep:1-21
        Guide: https://docs.bridgecrew.io/docs/bc_azr_iam_1
Check: CKV_AZURE_18: "Ensure that 'HTTP Version' is the latest if used to run the web app"
        PASSED for resource: Microsoft.Web/sites.Site
        File: example.bicep:1-21
        Guide: https://docs.bridgecrew.io/docs/bc_azr_networking_8
Check: CKV_AZURE_17: "Ensure the web app has 'Client Certificates (Incoming client certificates)' set"
        FAILED for resource: Microsoft.Web/sites.Site
        File: example.bicep:1-21
        Guide: https://docs.bridgecrew.io/docs/bc_azr_networking_7

                1  | resource Site 'Microsoft.Web/sites@2021-03-01' = {
                2  |   name: _siteName
                3  |   kind: kind
                4  |   location: location
                5  |   tags: _tags
                6  |   properties: {
                7  |     serverFarmId: ServerFarm.id
                8  |     httpsOnly: true
                9  |     clientAffinityEnabled: true
                10 |     reserved: _isReserved
                11 |     siteConfig: {
                12 |       minTlsVersion: '1.2'
                13 |       scmMinTlsVersion: '1.2'
                14 |       http20Enabled: true
                15 |       ftpsState: 'FtpsOnly'
                16 |     }
                17 |   }
                18 |   identity: {
                19 |     type: 'SystemAssigned'
                20 |   }
                21 | }

```

**Comment by holly79** [comment]
@gruebel : here the same, my bad. So I didn't see the PASSED. So from my site I think CKV_AZURE_14 is working
